.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ executors:
     working_directory: '~/go/src/github.com/influxdata/telegraf'
     resource_class: macos.m1.medium.gen1
     macos:
-      xcode: 14.2.0
+      xcode: 15.2.0
```

Comment:
Can we go all the way to 15.4? https://circleci.com/docs/testing-ios/#supported-xcode-versions-silicon